src/rules/javascript/content-scripts-file-absent.js

```diff
@@ -0,0 +1,31 @@
+import { normalizePath } from 'utils';
+import { CONTENT_SCRIPT_NOT_FOUND } from 'messages/javascript';
+
+export default {
+  create(context) {
+    const existingFiles = context.settings.existingFiles || {};
+    return {
+      CallExpression(node) {
+        if (node.callee.type === 'MemberExpression' &&
+          node.callee.property &&
+          node.callee.property.name === 'executeScript') {
+          node.arguments.forEach((arg) => {
+            if (arg.type !== 'ObjectExpression') return;
+
+            const fileProperty = arg.properties.find((prop) => prop.key && prop.key.name === 'file');
+            const fileValue = fileProperty && fileProperty.value.type === 'Literal' ? fileProperty.value.value : '';
+            if (fileValue.match(/content_scripts.*\.js/)) {
```

Comment:
I don't think that we should condition this validation on this regexp match, we can't assume that the filenames of the extension content scripts are going to contains "content_scripts" in their name.

Reply:
on the contrary, we should probably check if the file literal string is an empty string and produce a different validation error message (something like "unexpected empty filename" or similar error message).